assets/js/components/SapSystemDetails/GenericSystemDetails.test.jsx

```diff
@@ -73,4 +73,27 @@ describe('GenericSystemDetails', () => {
 
     expect(screen.getByText('ENSA version').nextSibling).toHaveTextContent('-');
   });
+
+  it('should render a cleanup button and correct health icon when absent instances exist', () => {
+    const sapSystem = keysToCamel(
```

Comment:
Why this `keysToCamel`? We don't serialize this way the redux state anymore
I guess you copied that from old tests, but I'm not even sure how they work.
I forgot to update those apparently...
We need to remove that function from the tests